templates/etc/systemd/system/tinc@.service.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
You should add commented out `{{ ansible_managed }}` at the start of the unit to indicate that it's managed by Ansible.